src/components/Map/Map.tsx

```diff
@@ -62,6 +70,22 @@ export default function Map({ countries, disputedAreas, ipcData }: MapProps) {
               toggleAlert={toggleAlert}
             />
           ))}
+      {selectedMapType === GlobalInsight.NUTRITION &&
+        countries.features &&
+        countries.features
+          .filter((countryData) => countryData.properties.interactive)
```

Comment:
right now every country is clickable and zoom with region split works for every country. you can filter out only clickable countries by checking if their id is in `nutritionData`
![image](https://github.com/user-attachments/assets/2af0675d-6d0d-464a-9187-93714fd574ec)